pages/cart/cart.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../../style.css">
+</head>
+
+<body>
+    <div class="navigation-container flex-hz">
+        <div class="navigation-item-1">
+            <h5 class="app-name-1">Naari <span class="app-name-2">Care</span></h5>
+        </div>
+        <div class="navigation-item-2">
+            <label class="search-bar">
+                <span><i class="fa fa-search"></i></span>
+                <input type="text" class="search-input" placeholder="search.." />
+            </label>
+        </div>
+        <div class="navigation-item-3 flex-hz">
+            <button class="btn btn-primary">Logout</button>
+            <i class="fa fa-shopping-cart nav-icons"><span class="badge-w-icon">1</span></i>
+            <i class="fa fa-heart nav-icons"><span class="badge-w-icon">10</span></i>
+        </div>
+    </div>
+    <h5 class="sub-heading">My Cart</h5>
+    <div class="cart-page-container">
+        <div class="cart-items-display">
+            <div class="card__container card-horizontal flex-hz">
+                <img class="card__image img-hz" src="../../images/prd-img.jpg" />
+                <div class="card-horizontal-column flex-vt flex-wrap">
+                    <h3 class="card__title">Olay Cream</h3>
+                    <p class="card__description">Lakme Absolute Perfect Radiance Brightening Day Cream 50 g, SPF
+                        30, Daily Illuminating Face Moisturizer for Glowing Skin - With Glycerin & Niacinamide
+                    </p>
+                    <div class="card__text-wrapper-1 flex-hz">
+                        <p class="card__text-primary">₹ 639</p>
+                        <p class="card__text-secondary">20% off</p>
+                    </div>
+                    <div class="card__text-wrapper-1 flex-hz">
+                        <p class="card__text-primary">Quantity :</p>
+                        <span class="flex-hz"><i class="fa fa-plus card-icon-plus"></i>
+                            <p class="card-text">5</p><i class="fa fa-minus card-icon-plus"></i>
+                        </span>
+                    </div>
+                    <div class="card-footer flex-hz">
+                        <button class="btn btn-primary card__btn-primary">ADD TO CART</button>
+                        <button class="btn btn-outline-primary card__btn-secondary">REMOVE FROM WISHLIST</button>
+                    </div>
+                </div>
+            </div>
+            <div class="card__container card-horizontal flex-hz">
+                <img class="card__image img-hz" src="../../images/prd-img.jpg" />
```

Comment:
alt??? The required alt attribute specifies an alternate text for an image, if the image cannot be displayed.